docs/example-scenario/apps/devops-dotnet-webapp.md

```diff
@@ -35,52 +36,44 @@ This scenario covers a DevOps pipeline for a .NET web application using Visual S
 
-Whilst this article focuses on Visual Studio Team Services, [Team Foundation Server][team-foundation-server] could be used as on premises substitute. Alternatively, you may also find a collection of technologies being used together for an Open Source development pipeline leveraging [Jenkins][jenkins-on-azure].
+While this article focuses on Azure DevOps, [Team Foundation Server][team-foundation-server] could be used as on-premises substitute. Alternatively, you could also use a set of technologies for an open source development pipeline using [Jenkins][jenkins-on-azure].
 
-From an Infrastructure as Code perspective, [Azure Resource Manager Templates][arm-templates] are included as part of the Azure DevOps project, but you could consider [Terraform][terraform] or [Chef][chef] if you have investments here. If you prefer an Infrastructure as a Service (IaaS) based deployment and require configuration management, then you could consider either [Azure Automation State Configuration][desired-state-configuration], [Ansible][ansible] or [Chef][chef].
+From an infrastructure-as-code perspective, [Azure Resource Manager Templates][arm-templates] are included as part of the Azure DevOps project, but you could consider [Terraform][terraform] or [Chef][chef]. If you prefer an infrastructure-as-a-service (IaaS)-based deployment and require configuration management, you could consider either [Azure Automation State Configuration][desired-state-configuration], [Ansible][ansible], or [Chef][chef].
 
-### Alternatives to Web App Hosting
+### Alternatives to Azure Web Apps
 
-Alternatives to hosting in Azure Web Apps:
+You could consider these alternatives to hosting in Azure Web Apps:
 
-* [VM][compare-vm-hosting] - For workloads that require a high degree of control, or depend on OS components / services that are not possible with Web Apps (for example, the Windows GAC, or COM)
-* [Container Hosting][azure-containers] - Where there are OS dependencies and hosting portability, or hosting density, are also requirements.
-* [Service Fabric][service-fabric] - A good option if the workload architecture is focused around distributed components that benefit from being deployed and run across a cluster with a high degree of control. Service Fabric can also be used to host containers.
-* [Serverless Azure functions][azure-functions] - A good option if the workload architecture is centered around fine grained distributed components, requiring minimal dependencies, where individual components are only required to run on demand (not continuously) and orchestration of components is not required.
+* [Azure Virtual Machines][compare-vm-hosting] &mdash; For workloads that require a high degree of control, or depend on OS components and services that are not possible with Web Apps (for example, the Windows GAC, or COM).
+* [Container Hosting][azure-containers] &mdash; if specific OS dependencies and hosting portability, or hosting density, are required.
```

Comment:
"Containers" is not really an alternative to "Web Apps" because App Service can [run containers](https://azure.microsoft.com/en-us/services/app-service/containers/).